### PR TITLE
Extract script runner to a separate type; fix work with env. variables

### PR DESCRIPTION
**Pull request by uthark**
Fixes https://github.com/keikoproj/upgrade-manager/issues/129

1. Extract work with scripts to a separate type.
2. Fix work with env. variables.
3. Simplify API by not requiring kubectl binary.

### CODE REVIEW

**Comment by uthark** [comment]
@eytan-avisror and/or @shrinandj please take a look.

File: controllers/script_runner.go [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+)
+
+const (
+	// KubeCtlBinary is the path to the kubectl executable
+	KubeCtlBinary = "/usr/local/bin/kubectl"
+	// ShellBinary is the path to the shell executable
+	ShellBinary = "/bin/sh"
+)
+
+type ScriptRunner struct {
```
Comment (shrinandj) [inline controllers/script_runner.go:35]: Awesome!

**Review by shrinandj — APPROVED** [review]

**Comment by shrinandj** [comment]
Looks great. Thanks for fixing these @uthark 

@eytan-avisror Feel free to merge once you've reviewed this as well.

**Comment by shrinandj** [comment]
@uthark , sorry we merged few other PRs recently that has resulted in a conflict here. Can you rebased and push again? We can merge and do a new build.

**Comment by uthark** [comment]
@shrinandj rebased.

**Pull request merged**